src/Surfnet/StepupMiddleware/GatewayBundle/Entity/InstitutionConfiguration.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
```

Comment:
strict types?

Reply:
That would be awesome, but as a rule of thumb, we only state this on new, or heavily refactored files. That way you can be sure every execution path is tested and verified to be  type strict/safe.

Reply:
Good call, fixed this for all new (non test) classes.